Dockerfile

```
@@ -1,10 +1,10 @@
-FROM matomo:4.14.2-apache
+FROM matomo:4.15.1-apache
 
 RUN set -ex && \
-  apt-get update && \
+  apt-get update -y && \
```

```suggestion
  apt-get update && \
```

There is no need as `update` operation does not expect any user input